lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -15,7 +15,7 @@ include_once 'dosomething_user.theme.inc';
  */
 function dosomething_user_preprocess_page(&$vars) {
   $school_api_endpoint = variable_get('dosomething_user_school_api_endpoint', 'http://lofischools.herokuapp.com/search');
-  $date_format = (dosomething_settings_get_geo_country_code() === 'US') ? 'MM/DD/YYYY' : 'DD/MM/YYYY';
```

Comment:
Yeah, I figured this has to do w/ how the validation and/or if user already entered date and it's repopulating the field etc... the data format thing is only for presenting _how_ to format it within the placeholder, but not actually dealing w/ the formatting, so figured it was best to leave the rest of that code as is :smile: